### DISCUSSION TITLE
Select functions not working

### DISCUSSION THREAD

**Discussion by Stewartgg32** [qarmin/czkawka#950] [Q&A]
Just installed the latest version on mac and am unable to use any of the select functions. 

I am sure it is a user issue but I am unable to find any way to get them to work and I have searched this board for an answer or similar issue with no luck. . 

When I select any option in that list it does nothing. I am able to select one by one. Thanks in advance. 

![image](https://user-images.githubusercontent.com/128245057/226119097-87879a54-ec06-494b-baec-7c3f3f02e887.png)

**Comment by Stewartgg32** [comment]
I also just recognized that when I attempt to change the check method or hash type it also doesnt work.

**Comment by Ech-io** [comment]
Same issue here, anyone had any luck getting it working? Thanks!

**Comment by David72645** [comment]
Same issue. Not user error I think.  I'm on an intel Mac, if that is important.
